File: empty_segments.py
import re

with open('index.html', 'r', encoding='utf-8') as f:
    text = f.read()

text = re.sub(
    r'<div class="segmented-control" id="inventory-locations">.*?</div>',
    '<div class="segmented-control" id="inventory-locations"></div>',
    text,
    flags=re.DOTALL
)

text = re.sub(
    r'<div class="segmented-control" id="receipt-location"[^>]*>.*?</div>',
    '<div class="segmented-control" id="receipt-location" style="margin-top: 4px;"></div>',
    text,
    flags=re.DOTALL
)

text = re.sub(
    r'<div class="segmented-control" id="form-location">.*?</div>',
    '<div class="segmented-control" id="form-location"></div>',
    text,
    flags=re.DOTALL
)

with open('index.html', 'w', encoding='utf-8') as f:
    f.write(text)

File: empty_segments.js
const fs = require('fs');
let text = fs.readFileSync('index.html', 'utf8');

text = text.replace(/<div class="segmented-control" id="inventory-locations">[\s\S]*?<\/div>/, '<div class="segmented-control" id="inventory-locations"><\/div>');
text = text.replace(/<div class="segmented-control" id="receipt-location"[^>]*>[\s\S]*?<\/div>/, '<div class="segmented-control" id="receipt-location" style="margin-top: 4px;"><\/div>');
text = text.replace(/<div class="segmented-control" id="form-location">[\s\S]*?<\/div>/, '<div class="segmented-control" id="form-location"><\/div>');

fs.writeFileSync('index.html', text, 'utf8');
